ui/src/app/app.component.scss

```diff
@@ -7,7 +7,7 @@
     width: 100%;
     position: fixed;
     display: none;
-    z-index: 200;
+    z-index: 1020;
```

Comment:
why not :p

Reply:
it's beacause the new dimmer from semantic has a z-index 1000